cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -105,7 +105,7 @@
             <div class="o-post-preview_image-container">
                 <img class="o-post-preview_image"
                      src="{{ photo.url }}"
-                     alt="{{ post.preview_image.alt if post.preview_image.alt else '' }}">
+                     alt="{{ photo.alt }}">
```

Comment:
We should eliminate that difference. You should always be able to call the context function and get the alt value if it exists, or '' if it doesn't.